src/polyfills/submit-event.ts

```diff
@@ -17,12 +17,23 @@ function clickCaptured(event: Event) {
 }
 
 (function() {
-  if ("SubmitEvent" in window) return
   if ("submitter" in Event.prototype) return
 
+  let prototype;
+  // Certain versions of Safari 15 have a bug where they won't
+  // populate the submitter. This hurts TurboDrive's enable/disable detection.
+  // See https://bugs.webkit.org/show_bug.cgi?id=229660
+  if ("SubmitEvent" in window && /Apple Computer/.test(navigator.vendor)) {
```

Comment:
Silly question, since Safar iOS is probably similar to desktop Safari, is there a chance this will need to be fixed when iOS 15 ships?

Reply:
Appears navigator.vendor can only be one of:
`"Google Inc."`, `"Apple Computer Inc."`, or `""`

So I think we should be okay (you also tested it as well, thank you!)